### PR TITLE
Control Spec update CloudService ,CosmosDB

### PR DESCRIPTION
**Pull request by Aboli-msft**

### CODE REVIEW

File: Control coverage/Feature/CloudService.md [hunk 1]
```diff
@@ -4,3 +4,5 @@
...
+| <b>ControlId:</b><br>Azure_CloudService_DP_DontAllow_HTTP_Access_InputEndpoints<br><b>DisplayName:</b><br>Encrypt data in transit for Cloud service role <br><b>Description: </b><br> Cloud Service must only be accessible over HTTPS. Enable https for InputEndpoints.| <b> ARM API to get Cloud Service roles details: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ClassicCompute/domainNames/{cloudServiceName}/slots/{slotName}/roles/{roleName}?api-version=2016-04-01  <br><b>Properties:</b><br> name: 'properties.inputEndpoints.protocol| <b>Passed: </b><br>No active non SSL enabled input endpoints for all the roles in cloud service. <br><b>Failed: </b><br>Active non SSL enabled input endpoints are present for any of the roles in cloud service. |
```
Comment (TarunKrShukla) [inline Control coverage/Feature/CloudService.md:7]: Looks like there is typo in Properties [name: 'properties.inputEndpoints.protocol]

File: Control coverage/Feature/CloudService.md [hunk 2]
```diff
@@ -4,3 +4,5 @@
...
+| <b>ControlId:</b><br>Azure_CloudService_SI_Auto_OSUpdate<br><b>DisplayName:</b><br>Set automatic update for Cloud Service OS version<br><b>Description: </b><br> Set automatic update for Cloud Service OS version.| <b> ARM API to get Cloud Service extension details: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ClassicCompute/domainNames/{cloudServiceName}/slots/{slotName}/roles?api-version=2016-04-01<br><b>Properties:</b><br> name: properties.slotType,properties.configuration.osVersion | <b>Passed: </b><br> Cloud service is enabled with automatic OS updates for all slots with slotType as 'Role'. <br><b>Failed: </b><br> Cloud service is not set up for automatic OS updates for any of the slots with slotType as 'Role'. |
```
Comment (TarunKrShukla) [inline Control coverage/Feature/CloudService.md:8]: If there are multiple properties please use space after each property.

File: Control coverage/Feature/CloudService.md (hunk 1 above)
Comment (TarunKrShukla) [inline Control coverage/Feature/CloudService.md:7]: Also in API we might want to add API to get slot or roles.

File: Control coverage/Feature/CloudService.md (hunk 2 above)
Comment (TarunKrShukla) [inline Control coverage/Feature/CloudService.md:8]: Also in API we might want to add API to get slot or roles.

File: Control coverage/Feature/CosmosDB.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+| <b>ControlId:</b><br>Azure_CosmosDB_Enable_Adv_Threat_Protection<br><b>DisplayName:</b><br>Enable Threat detection for CosmosDB database<br><b>Description: </b><br> Threat Protection for Azure Cosmos DB provides an additional layer of security intelligence that detects unusual and potentially harmful attempts to access or exploit Azure Cosmos DB accounts.| <b> ARM API to get CosmosDB resources in a subscription  </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/databaseAccounts?api-version=2019-08-01  <br><b>Properties:</b><br> name: 'properties.EnabledApiTypes'<br><b>ARM API to get advanced threat protection settings</b><br> /{ResourceId}/providers/Microsoft.Security/advancedThreatProtectionSettings/current?api-version=2017-08-01-preview<br><b>Properties:</b><br> name: 'properties.isEnabled'|<b>Passed: </b><br>  Advanced Threat Protection is enabled for CosmosDB. <br><b>Failed: </b><br> Advanced Threat Protection is not enabled for CosmosDB.<br><b>NotApplicable </b><br>Advanced Threat Protection is not available for the enabled API option(s) |
```
Comment (TarunKrShukla) [inline Control coverage/Feature/CosmosDB.md:9]: In property column 'name:' can be removed for all controls

**Review by TarunKrShukla — CHANGES_REQUESTED** [review]

File: Control coverage/Feature/CosmosDB.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+## Cloud Service
```
Comment (TarunKrShukla) [inline Control coverage/Feature/CosmosDB.md:1]: This heading should be CosmosDB

Comment (TarunKrShukla) [reply Control coverage/Feature/CosmosDB.md:1]: Also in main page link need to be added for CosmosDB, as it's a new service.

**Comment by Aboli-msft** [comment]
PR no more required . Control spec updated with latest format

**Pull request closed without merging**